Blank space bellow title after written or vocal command in the "Doorhanger"

### Prerequisites:
Mic permissions are enabled.

### STR:
1. Use the shortcut or click on the mic icon from the browser toolbar.
2. Say or write a "Search" command, e.g. "Search lion".

### Expected result: 
The "Doorhanger" doesn't have display issues.

### Actual result:
Bellow the searched title there is a blank space.

### Notes:
Reproduced on Mac 10.14.6 and Win10x64 with Firefox Nightly 71.0a1 (64-bit).

![80](https://user-images.githubusercontent.com/54401091/65874714-dcc0e100-e38e-11e9-985e-cc9e29b66120.gif)

I'm having trouble starting the project with **npm start** . The project is installed properly, but every time I try to start it, I get an error message such as :
 `npm ERR! Failed at the firefox-voice@0.8.0 start-extension script.
Error: not found: firefox-nightly
    at F (/mnt/c/Users/Thomas/projects/firefox-voice/node_modules/fx-runner/node_modules/which/which.js:63:19)
    at E (/mnt/c/Users/Thomas/projects/firefox-voice/node_modules/fx-runner/node_modules/which/which.js:72:29)
    at /mnt/c/Users/Thomas/projects/firefox-voice/node_modules/fx-runner/node_modules/which/which.js:81:16
    at /mnt/c/Users/Thomas/projects/firefox-voice/node_modules/fx-runner/node_modules/isexe/index.js:44:5
    at /mnt/c/Users/Thomas/projects/firefox-voice/node_modules/fx-runner/node_modules/isexe/access.js:8:5
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)`

I'm using Win10 (with WSL - Ubuntu) and having Firefox and Nightly both installed on Windows. I have tried to change the path (`$FIREFOX`) at the "start-extension" script in the **package.json** file, to access the firefox.exe in Mozilla Firefox's package, but without any success.

--- a/extension/popup/popup.css
+++ b/extension/popup/popup.css
@@ -62,6 +62,11 @@ body,
   display: block;
 }
 
+.success.hasCard #text-input,
+.success.hasCard #error-message {
+  display: none;
+}
+
 .success.hasCard #zap-wrapper {
   transition-property: max-height;
   max-height: 0;

--- a/extension/popup/ui.js
+++ b/extension/popup/ui.js
@@ -181,7 +181,11 @@ this.ui = (function() {
   exports.showCard = function showCard(data) {
     document.querySelector("#popup").classList.add("hasCard");
     document.querySelector("#card-header").textContent = data.Heading;
-    document.querySelector("#card-image > img").src = data.Image;
+    if (data.Image) {
+      document.querySelector("#card-image > img").src = data.Image;
+    } else {
+      document.querySelector("#card-image").style.display = "none";
+    }
     document.querySelector("#card-summary").textContent = data.AbstractText;
     document.querySelector("#card-source-link").textContent =
       data.AbstractSource;

